Dockerfile

```diff
@@ -20,4 +20,6 @@ RUN pip install awsebcli
 ADD ./ssh root/.ssh
 RUN chmod 700 root/.ssh/gitlab_rsa
 
+RUN npm i -g yarn@v1.3.2

```

Comment:
Should we add a follow-up issue to find a way to use non npm yarn? My understanding is that this isn't the recommended way. I think it's extremely low priority but I'd feel better tracking it (or in case someone looking at our repo wanting to contribute feels like taking it on).

Reply:
I agree, but I think the yarn that comes with this image was itself installed using npm, which is likely why this is the only thing that worked for Tocco.

I've created #4 for us to come back with a better solution when we're unblocked